exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
+    Args:
+        file_path (string): path to atm information file.
+
+    Return:
+        A dictionary, the key is each users id.
+        the value is a list of user's password and balance.
```

Comment:
For cases like this we usually use tuples instead of lists. Tuples are immutable (cannot be overwritten) so we use them for cases where we aren't changing (ie. mutating) the information inside of them.

All you will need to do to change the brackets you used to create the list to parenthesis to create a tuple.

read more here especially number 11. https://data-flair.training/blogs/python-tuples-vs-lists/

Reply:
but i need to change the vlues in it, when the user want to change password or deposit or withdraw money. i think that if i will use a tuple, i will need to cahnge it to list, change the thing the user want and then change it back.